frontend/src/metabase/lib/urls/dataApps.ts

```diff
@@ -0,0 +1,13 @@
+import slugg from "slugg";
+
+import { DataApp } from "metabase-types/api";
+
+import { appendSlug } from "./utils";
+
+function dataAppInternal(app: DataApp) {
+  return appendSlug(`/a/${app.id}`, slugg(app.collection.name));
+}
+
+export function dataApp(app: DataApp) {
+  return dataAppInternal(app);
+}
```

Comment:
Eventually, we want apps to be available under their own unique URLs (e.g. under a custom domain), so for now there's a notion of an "internal" URL which is `/a/:id`. Not using `/app` for now as it conflicts with our current setup where we're serving static files under the `/app` path